devices/motorola-potter/default.nix

```diff
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+
+{
+  mobile.device.name = "motorola-potter";
+  mobile.device.identity = {
+    name = "Moto G5 Plus";
+    manufacturer = "Motorola";
+  };
+
+  mobile.hardware = {
+    soc = "qualcomm-msm8953";
+    ram = 1024 * 3;
+    screen = {
+      width = 1080; height = 1920;
+    };
+  };
+
+  mobile.boot.stage-1 = {
+    kernel.package = pkgs.callPackage ./kernel { };
+  };
+
+  mobile.device.firmware = pkgs.callPackage ./firmware {};
+
+  hardware.firmware = [
+    (config.mobile.device.firmware.override {
+      modem = ../../vendor/modem;
```

Comment:
Probably won't work for other people :)

Reply:
no, you're right. will delete before merge